addons/arsenal/stringtable.xml

```diff
@@ -745,6 +752,7 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_exportedClassnameText">
             <English>Exported class name to clipboard</English>
+            <German>Exportiere Class-Name in Zwischenablage kopieren</German>
```

Comment:
Klassenname

Reply:
Der Satz ergibt so wenig Sinn (zwei Verben "Exportiere" und "kopieren"). Vielleicht "Exportierte Klassennamen in Zwischenablage kopieren" oder "Klassenname in Zwischenablage exportieren" ?

Reply:
I think the text is a status message for something that happened in the past.
"Exported class name to clipboard" -> "Der Klassenname wurde in die Zwischenablage kopiert"